node/url.ts

```diff
@@ -325,6 +332,7 @@ export default {
   format,
   fileURLToPath,
   pathToFileURL,
+  toPathIfFileURL,
```

Comment:
Is this used somewhere?

Reply:
it's imported in `node/internal/fs/utils.js`